.devcontainer/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN \
   libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 \
   libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 \
   libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates \
-  fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget \
+  fonts-liberation libayatana-appindicator1 libnss3 lsb-release xdg-utils wget \
```

Comment:
can you add `libsecret-1-0` to this list as well? I just tested this out and looks like that one is missing for the full-stack-tests to run puppeteer properly.

Reply:
Added.

Reply:
thanks!